src/component/App.js

```diff
@@ -1,12 +1,46 @@
 import './App.css';
-import DataTable from './DataTable'
+
+import firebase from 'firebase/app';
+import 'firebase/firestore';
+import 'firebase/auth';
+
+import { useCollectionData } from 'react-firebase-hooks/firestore';
+
+firebase.initializeApp({
+    apiKey: "AIzaSyBe9RfBFcXjrGtQ7jVxtrxSOWhqf28DHYk",
+    authDomain: "gwwc-data-table.firebaseapp.com",
+    databaseURL: "https://gwwc-data-table.firebaseio.com",
+    projectId: "gwwc-data-table",
+    storageBucket: "gwwc-data-table.appspot.com",
+    messagingSenderId: "924988001345",
+    appId: "1:924988001345:web:655254cb48b7162b6f4ac0",
+    measurementId: "G-TC4E5JES7F"
+})
+
+const firestore = firebase.firestore();
```

Comment:
The code is trying to access **firestore** while we are using **realtime database**.
Also we don't need to use 'auth', the read access is public now - see 

Please try the code below to init the realtime database:
```
import firebase from 'firebase/app';
import 'firebase/database';

firebase.initializeApp({
    apiKey: "AIzaSyBe9RfBFcXjrGtQ7jVxtrxSOWhqf28DHYk",
    authDomain: "gwwc-data-table.firebaseapp.com",
    databaseURL: "https://gwwc-data-table.firebaseio.com",
    projectId: "gwwc-data-table",
    storageBucket: "gwwc-data-table.appspot.com",
    messagingSenderId: "924988001345",
    appId: "1:924988001345:web:655254cb48b7162b6f4ac0",
    measurementId: "G-TC4E5JES7F"
})

const database = firebase.database();
```